locale/en/docs/guides/nodejs-docker-webapp.md

```diff
@@ -110,13 +110,15 @@ COPY package*.json ./
 
 RUN npm install
```

Comment:
Should also be changed here.

Reply:
NPM CI is only required for production builds not the normal build